internal/cli/streams/instance/create.go

```diff
@@ -47,6 +48,11 @@ func (opts *CreateOpts) Run() error {
 	streamProcessor.Name = &opts.name
 	streamProcessor.GroupId = &opts.ProjectID
 	streamProcessor.DataProcessRegion = atlasv2.NewStreamsDataProcessRegion(opts.provider, opts.region)
+	if opts.tier != "" {
```

Comment:
if tier isn't passed in - we want to default to SP30 right?

Reply:
That's a good question. The endpoint this calls defaults to SP30. I wonder which is cleaner, the client picking a default or the server.

Reply:
Oh yeah the server will need the default regardless, so I reckon this is good and we shouldn't need to add it here as well.

Reply:
Had a chat with Nathan, ended up adding the default behaviour: [df8d5c2](https://github.com/mongodb/mongodb-atlas-cli/pull/2839/commits/df8d5c23b9f4736fe11de5ae054df1b209a6c150)

Reply:
Reason for having both; users could call the atlas api without using the cli.

Reply:
this will never be empty since you are setting it with 
```go
cmd.Flags().StringVar(&opts.tier, flag.Tier, "SP30", usage.StreamsInstanceTier)
```